kernel/cifsd/Makefile

```diff
@@ -0,0 +1,52 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cifsd
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/cifsd-team/cifsd.git
+PKG_SOURCE_DATE:=2019-05-31
+PKG_SOURCE_VERSION:=e0804cb48a36054c0b2d392d3dabd845c9be2ad8
+PKG_MIRROR_HASH:=72797679635741ff88ded72d9c2215dc51ba15d197536bfdf1592225da11f6bd
+
+PKG_MAINTAINER:=Andy Walsh <andy.walsh44+github@gmail.com>
+PKG_LICENSE:=GPL-2.0
```

Comment:
License is not corrected according to [SPDX](http://SPDX.org/licenses).

Reply:
fixed